src/static/js/ours/client.js

```diff
@@ -77,6 +77,9 @@ CODALAB.api = {
     re_run_phase_submissions: function (phase_pk) {
         return CODALAB.api.request('GET', `${URLS.API}phases/${phase_pk}/rerun_submissions/`)
     },
+    get_competition_files: pk => {
```

Comment:
did we need to add this because it was called directly before, or something?

Reply:
because we don't need the files if we aren't admin, so didn't put them on a serializer. plus it allows up to update _just_ the files after creating a dump, rather than reloading the whole competition. Or at least that was my logic.

Reply:
right, sorry, made this comment before reading your slack msg, ty!!!